qiita_db/portal.py

```diff
@@ -139,7 +139,7 @@ def delete(portal):
                 " Cannot delete portal '%s', analyses still attached: %s" %
                 (portal, ', '.join([str(a[0]) for a in analyses])))
 
-        # Remove portal and default analyses for all users
+        # Remove portal and default analyses for all users, unlink others
```

Comment:
Minor comment, but I just noticed that `pid` is being used anywhere in the SQL below.